src/components/Profile/History/PoolShares/Liquidity.tsx

```diff
@@ -14,38 +13,22 @@ export function Liquidity({
   row: AssetPoolShare
   type: string
 }) {
-  let price = '0'
   let liquidity = '0'
```

Comment:
guess it works as it was done quickly originally but proper React way would be to put this all into `useState` and `useEffect` to minimize recalculations

Reply:
true 😞